test/Closures/rlexe.xml

```diff
@@ -158,4 +158,10 @@
       <tags>BugFix</tags>
     </default>
   </test>
+  <test>
+    <default>
+      <files>bug_OS_9008744.js</files>
+      <tags>BugFix,exclude_dynapogo</tags>
```

Comment:
I agree. And you're right we haven't had any recently, but I'd say that's mostly because we haven't been doing much language work. Enabling these runs is non-trivial I'm afraid because of a ton of tests that need to be examined and excluded.